README.md

```diff
@@ -47,33 +32,60 @@ As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
+### Technologies Used:
+* [dotenv](https://github.com/bkeepers/dotenv) - loads ENV variables
+* [ngrok](https://ngrok.com/) - A secure tunnel to my localhost
+* [Pry](http://pryrepl.org/) - An awesome REPL for Ruby
+* [Ruby](https://www.ruby-lang.org/en/) - Programming Language
+* [Sinatra](http://sinatrarb.com/) - Domain Specific Language for Web Apps
+* [Twilio-Ruby](https://www.twilio.com/docs/libraries/ruby) - API that allows user to send and receive texts.
+* [VCR](https://github.com/vcr/vcr) - Records HTTP interactions for tests
+
+### Launch Program:
+```zsh
+$ pry -r './lib/takeaway.rb'
+[1] pry(main)> russian = Menu.new
+=> #<Menu:0x00007fde4d5c0a48
+ @current_menu=
+  {"pelmeni"=>6.99,
+   "borscht"=>5.99,
+   "tvarog"=>3.99,
+   "olivier"=>4.99,
+   "salat"=>1.99,
+   "blinchiki"=>9.99}>
+[2] pry(main)> shop = Takeaway.new
+[3] pry(main)> shop.order("pelmeni", 2)
+=> "2x - Pelmeni added to basket. Your current total is: £13.98"
+[5] pry(main)> shop.order_confirmation
+=> "Spacibo! You will receive a text soon confirming your order. Next time, try ordering over text message!"
+```
+#### Photo of text: 
+![photo of text](https://i.imgur.com/kbnSzxZ.jpg?1)
+
+**I was able to integrate the bonus challenge** into my program using ngrok and Sinatra, allowing you to order food by texting the number. 
+
+Photos:
+
+![placing order](https://i.imgur.com/P6ojEIp.png?1)![choosing food](https://i.imgur.com/vJtBmq0.png?4)![getting total](https://i.imgur.com/i1cqSZ1.png?1)![final confirm](https://i.imgur.com/RgC6B95.png?1)
+
+### My Process
```

Comment:
Some of your commits are nice and small, sometimes a commit included a few changes. Try to keep committing after every change as much as possible.